tools/ci_build/github/linux/ort_minimal/build_minimal_ort_android_baseline_and_report_bin_size.sh

```diff
@@ -30,7 +30,7 @@ python3 /onnxruntime_src/tools/ci_build/build.py \
 
 # set current size limit to 1165KB.
```

Comment:
```suggestion
# set current size limit to 1175KB.
```
or replace this comment with something like `BINARY_SIZE_LIMIT_IN_BYTES=1175000` and use that below

Reply:
Will fix these comments in my next PR